src/System.Windows.Forms/src/System/Windows/Forms/ListViewItem.ListViewItemDetailsAccessibleObject.cs

```diff
@@ -112,9 +161,9 @@ private int GetFakeSubItemIndex(ListViewSubItem.ListViewSubItemAccessibleObject
                 return -1;
             }
 
-            internal override Rectangle GetSubItemBounds(int subItemIndex)
-                => _owningListView.IsHandleCreated
-                    ? _owningListView.GetSubItemRect(_owningItem.Index, subItemIndex)
+            internal override Rectangle GetSubItemBounds(int accessibleChildIndex) =>
+                _owningListView.IsHandleCreated
+                    ? _owningListView.GetSubItemRect(_owningItem.Index, ConvertToSubItemIndex(accessibleChildIndex))
```

Comment:
[nit] I believe that in the multiline expressions `=>` should be on the new line
```suggestion
            internal override Rectangle GetSubItemBounds(int accessibleChildIndex)
                => _owningListView.IsHandleCreated
                    ? _owningListView.GetSubItemRect(_owningItem.Index, ConvertToSubItemIndex(accessibleChildIndex))
```